### PR TITLE
feat(tailwind): introduce container queries

### PR DESCRIPTION
**Pull request by fulopdaniel**
[CSS variables are not supported in media queries](https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries). Because of this I did not add the CSS variables for each breakpoint, as I don't think they could be consumed either by us or our partners.

We can enable our partners by providing a shared tailwind config that they can reuse.

### CODE REVIEW

File: .changeset/curvy-cheetahs-collect.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/guideline-blocks-settings": patch
+---
+
+feat(tailwind): introduce container queries
```
Comment (ragi96) [inline .changeset/curvy-cheetahs-collect.md:5]: Here add how to use it

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -245,19 +245,19 @@ importers:
...
         specifier: ^7.0.0
-        version: 7.0.0(@dnd-kit/core@6.1.0)(react@18.2.0)
+        version: 7.0.0(@dnd-kit/core@6.1.0(react-dom@18.2.0(react@18.2.0))(react@18.2.0))(react@18.2.0)
       '@dnd-kit/sortable':
         specifier: ^8.0.0
-        version: 8.0.0(@dnd-kit/core@6.1.0)(react@18.2.0)
+        version: 8.0.0(@dnd-kit/core@6.1.0(react-dom@18.2.0(react@18.2.0))(react@18.2.0))(react@18.2.0)
       '@frontify/app-bridge':
         specifier: ^3.0.0 || ^4.0.0-alpha.0
-        version: link:../app-bridge
```
Comment (ragi96) [inline pnpm-lock.yaml:257]: This here is off :D

File: .changeset/curvy-cheetahs-collect.md (hunk 1 above)
Comment (fulopdaniel) [reply .changeset/curvy-cheetahs-collect.md:5]: Added

File: pnpm-lock.yaml (hunk 2 above)
Comment (fulopdaniel) [reply pnpm-lock.yaml:257]: Interesting, but `pnpm i` generates my version.

Comment (ragi96) [reply pnpm-lock.yaml:257]: Most likely you did `pnpm add @frontify/app-bridge@workspace:^ --filter {packages/guideline-blocks-settings}` to build and test etc

Comment (ragi96) [reply pnpm-lock.yaml:257]: Not 100% sure just feels odd, but yeah I don't think it matters

File: .changeset/curvy-cheetahs-collect.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+`@frontify/guideline-block-settings/tailwind` can now be used as a tailwind preset in your tailwind.config.ts file to enable container queries. On how to use them refer to tailwind documentation.
```
Comment (ragi96) [inline .changeset/curvy-cheetahs-collect.md:6]: maybe link the doc in here?

File: .changeset/curvy-cheetahs-collect.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+"@frontify/guideline-blocks-settings": patch
```
Comment (ragi96) [inline .changeset/curvy-cheetahs-collect.md:2]:
```suggestion
"@frontify/guideline-blocks-settings": minor
```

**Review by ragi96 — COMMENTED** [review]
Do you have a block branch to test it quickly? So I can play around with it :D

File: .changeset/curvy-cheetahs-collect.md (hunk 4 above)
Comment (fulopdaniel) [reply .changeset/curvy-cheetahs-collect.md:2]: is there a guideline to what it takes for a change to be minor instead of patch?

Comment (ragi96) [reply .changeset/curvy-cheetahs-collect.md:2]: Basically this one: https://semver.org/

**Comment by fulopdaniel** [comment]
> Do you have a block branch to test it quickly? So I can play around with it :D

https://github.com/Frontify/guideline-blocks/tree/poc/container-query

This should work, added test query to text block.

File: .changeset/curvy-cheetahs-collect.md (hunk 3 above)
Comment (fulopdaniel) [reply .changeset/curvy-cheetahs-collect.md]: added

**Review by ragi96 — APPROVED** [review]
LGTM

**Pull request merged**